.nvmrc

```diff
@@ -0,0 +1 @@
+v8.16.0
```

Comment:
Node is on 13 now. This should probably be up to 11 at least. 12-13 add some new things that might cause breakage but 8 is pretty old.

Reply:
Guess it isn't THAT old:

Node.js 8.16.0 | Carbon | 2019-04-16
-- | -- | --

https://nodejs.org/en/download/releases/

Reply:
Whoops. Great catch @sherakama ! Thank you :)